src/test/java/core/basesyntax/ProcessServiceTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.ProcessingService;
+import core.basesyntax.service.impl.ProcessingServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operations.BalanceHandler;
+import core.basesyntax.strategy.operations.OperationHandler;
+import core.basesyntax.strategy.operations.PurchaseHandler;
+import core.basesyntax.strategy.operations.ReturnHandler;
+import core.basesyntax.strategy.operations.SupplyHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProcessServiceTest {
+    private List<String> emptyTestList;
+    private List<String> firstTestList;
+    private List<String> secondTestList;
+    private List<String> thirdTestList;
+    private OperationStrategy strategy;
+    private ProcessingService processingService;
+
+    @Before
+    public void setUp() {
+        Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(Operation.BALANCE, new BalanceHandler());
+        operationHandlerMap.put(Operation.PURCHASE, new PurchaseHandler());
+        operationHandlerMap.put(Operation.RETURN, new ReturnHandler());
+        operationHandlerMap.put(Operation.SUPPLY, new SupplyHandler());
+        strategy = new OperationStrategyImpl(operationHandlerMap);
+        processingService = new ProcessingServiceImpl();
+        firstTestList = List.of(
+                "b,banana,20", "b,apple,100", "b,oranges,240",
+                "s,banana,100", "p,banana,13", "r,apple,10", "p,oranges,20",
+                "p,apple,20", "p,banana,5", "s,banana,50", "r,oranges,50");
+        secondTestList = List.of("b,banana,20", "b,apple,100", "b,oranges,240");
+        thirdTestList = List.of("b,banana,20", "p,banana,20", "p,banana,1");
+        emptyTestList = List.of();
```

Comment:
This looks like creating redundant arraylists, because you don't use each of them in your tests at the same time